src/app/manager-dashboard/manager-sync.component.ts

```diff
@@ -44,21 +69,8 @@ export class ManagerSyncComponent implements OnInit {
   }
 
   replicatorList() {
-    // List of replicators to push to parent planet
-    const pushList = [
-      { db: 'courses_progress' },
-      { db: 'feedback' },
-      { db: 'login_activities' },
-      { db: 'ratings' },
-      { db: 'resource_activities' }
-    ];
-    // List of replicators to pull from parent
-    const pullList = [
-      { db: 'feedback', selector: { source: this.userService.getConfig().code } },
-      { db: 'notifications', selector: { target: this.userService.getConfig().code } }
-    ];
     const addType = (type) => (val) => ({ ...val, type });
-    return pushList.map(addType('push')).concat(pullList.map(addType('pull')));
+    return this.pushList.map(addType('push')).concat(this.pullList.map(addType('pull')));
```

Comment:
You should refactor to use `getDefaultList` rather than duplicate this line.  `getDefaultList` should use the addType function on line 72.  You can still keep these in the `deleteArray` list by using `defaultList.findIndex(...)` rather than `indexOf`